src/uhttp.c

```diff
@@ -320,7 +325,48 @@ static int write_text_line(HTTP_CLIENT_HANDLE_DATA* http_data, const char* text_
         result = 0;
         if (http_data->trace_on)
         {
-            LOG(AZ_LOG_TRACE, LOG_LINE, "%s", text_line);
+            char* authStart = strstr(text_line, AUTH_HEADER);
+            char* proxyAuthStart = strstr(text_line, PROXY_AUTH_HEADER);
+            char* authEol = strstr(authStart, HEADER_ENDING);
+            char* proxyAuthEol = strstr(proxyAuthStart, HEADER_ENDING);
+            if (authStart == NULL && proxyAuthStart == NULL)
+            {
+                LOG(AZ_LOG_TRACE, LOG_LINE, "%s", text_line);
+            }
+            else if (authStart != NULL && proxyAuthStart == NULL)
+            {
+                if (authEol != NULL)
+                {                                                                 
+                    LOG(AZ_LOG_TRACE, LOG_LINE, "%.*s ************** %.*s", (int)(authStart  - text_line), text_line, (int)(strlen(text_line) - (authEol - text_line)), authEol);
+                }
+            }
+            else if (proxyAuthStart != NULL && authStart == NULL)
```

Comment:
Since `Authorization:` is in `Proxy-Authorization:`, if proxyAuthStart != NULL, won't authStart always != NULL?

Reply:
Great catch, I will fix that.